packages/synapse-interface/pages/bridge/index.tsx

```diff
@@ -739,14 +768,30 @@ const BridgePage = ({
           duration: 10000,
         })
 
+        // TODO: are these values different from the initiator?
```

Comment:
I think this would only be different if there was a large delay between the initiation and completion in wallet and the quote re-updated. I think we don't need to re-use the values in this event since the assumption that they're the same is good.